perfkitbenchmarker/providers/gcp/gcp_spanner.py

```diff
@@ -45,8 +45,8 @@ def __init__(self, name, desp, nodes, config, database, ddl):
     self._database = database
     self._ddl = ddl
 
-    # Cloud Spanner is not using the following common flags.
-    self.project = None
+    # Cloud Spanner does not explicitly set the following common flags.
+    self.project = util.GetDefaultProject()
```

Comment:
https://cloud.google.com/spanner/docs/gcloud-spanner#set_a_default_project

How do you specify a project different from default one?